packages/front-end/components/Features/ConditionDisplay.tsx

```diff
@@ -5,7 +5,16 @@ import InlineCode from "../SyntaxHighlighting/InlineCode";
 import stringify from "json-stringify-pretty-compact";
 import { useMemo } from "react";
 
-function operatorToText(operator: string): string {
+function operatorToText(operator: string, field: string): string {
+  if (field === "current_datetime") {
```

Comment:
Change this so that it's not hard-coded to `current_datetime`. Maybe something like we did for $inGroup and $notInGroup?